src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
-'use strict';
+"use strict";
 
-// write code here
+document.querySelectorAll(".tree li").forEach((li) => {
+  const span = document.createElement("span");
+
+  li.prepend(span);
+  span.appendChild(span.nextSibling);
+});
+
+document.querySelector(".tree").addEventListener("click", function (e) {
```

Comment:
Click eventListener is added only to first '.tree' element, it won't work for other '.tree' elements if there are multiple on the page.